documentation/library_guide/onedpl_gsg.rst

```diff
@@ -44,6 +44,44 @@ All |onedpl_short| header files are in the ``oneapi/dpl`` directory. Use ``#incl
 To use tested C++ standard APIs, you need to include the corresponding C++ standard header files
 and use the ``std`` namespace.
 
+CMake Support
+-------------
+`CMake <https://cmake.org/cmake/help/latest/index.html>`_ generates build scripts which can then be used to build and link your application.  |onedpl_short| can be added to your project via CMake.  
+
+A simple example for Linux is provided below.  For more detailed usage and options, please look to the `CMake Support Page <https://www.intel.com/content/www/us/en/docs/onedpl/developer-guide/current/cmake-support.html>`_.

```

Comment:
There is another link to oneDPL library guide: https://oneapi-src.github.io/oneDPL/
Unfortunately, I do not know which link should be in each `onedpl_gsg.rst`.

Reply:
Yes, I was unsure of that as well.  Perhaps @dcbenito can provide some guidance on this.

Reply:
The issue is that both links are, or will be, correct. The github.io is the open source version and the Intel version is https://www.intel.com/content/www/us/en/docs/onedpl/developer-guide/current/overview.html. We may be able to create a "condition" using the "variables.txt" file, but I seem to remember that the formatting can have issues with links used in conditions. The other way of doing it would be to add both links, notating that one is open source and one is for the Intel product.

Reply:
Am I correct in thinking that one of the `onedpl_gsg.rst` is meant to go alongside the open source version, and the other is to go along side the intel product?  If so, can we just make the links point to their matching spots?

Reply:
You are correct, and that is a great solution.

Reply:
OK,  I've made this change.  Thanks